magento/magento2
# Fix #21648 Checkout Agreements checkbox missing asterisk

### Description (*)



### Fixed Issues (if relevant)

1. magento/magento2#21648 : missing asterisk in checkout Agreements


### Manual testing scenarios (*)
Create Terms and Conditions agreement
Enable it with "manual" setting so checkbox would be shown
Go to checkout
Expected result (*)
Agreement checkbox label should have red asterisk as it's required to check it

Actual result (*)
Asterisk is missing and after pressing place order gives message that field is required
### Contribution checklist (*)
